lib/utils/azure_storage.ts

```diff
@@ -0,0 +1,48 @@
+import * as azureStorage from "azure-storage";
```

Comment:
would be nice a block comment explaining what this file provides (e.g. helpers for the Azure Storage service)

Reply:
ok